presto-main/src/main/java/io/prestosql/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -477,6 +479,23 @@ private void printWindowOperatorStats(int indent, WindowOperatorStats stats)
         output.append('\n');
     }
 
+    private static String formatDoubleAsCpuCost(double value)
+    {
+        return formatDoubleAsDataSize(value).replaceAll("B$", "");
```

Comment:
This is not ideal as it uses "data size", which is completely unrelated. Although that class/format doesn't change much, it makes this code brittle. But it will do for now. We can later have a dedicated formatter for cost units.